doc/src/for-hevm-developers.md

```diff
@@ -55,6 +55,28 @@ Similarly, the `maxSuccess` value can be modified for a single test. The followi
 testProperty <property_name> $ withMaxSuccess 20 $ ...
 ```
 
+## Running benchmarks
+
+You can also measure and compare the performance across hevm versions using the benchmarks.
+`bench-perf` focuses on concrete execution performance, and `bench` is aimed at
```

Comment:
This new code is about `bench-perf`, right? It's about concrete execution.
Do we have documentation on `bench`?